use-cases/eurac/README.md

```diff
@@ -0,0 +1,99 @@
+# EURAC use case
+
+## Installation
+
+First, make sure to install itwinai from this branch!
+Use the [developer installation instructions](https://github.com/interTwin-eu/itwinai/tree/usecase_eurac?tab=readme-ov-file#installation-for-developers).
+
+Then:
+
+```bash
+pip install -r requirements.txt
+```
+
+## Interactive session on SLURM
+
+Allocate 4 GPUs on a compute node and run distributed algorithms:
+see "[Distributed training on a single node (interactive)](https://github.com/interTwin-eu/itwinai/tree/main/tutorials/distributed-ml/torch-tutorial-0-basics#distributed-training-on-a-single-node-interactive)."
+
+```bash
+
+salloc --partition=batch --nodes=1 --account=intertwin  --gres=gpu:4 --time=1:59:00
+
+srun --jobid XXXX --overlap --pty /bin/bash
+
+ml --force purge
+
+ml Stages/2024 GCC OpenMPI CUDA/12 MPI-settings/CUDA Python HDF5 PnetCDF libaio mpi4py
+
+source ../../../hython-dev/bin/activate
+
+torchrun --standalone --nnodes=1 --nproc-per-node=gpu dist-train.py
+
+```
+
+## Training
+
+You can run the whole pipeline in one shot, including dataset generation, or you can
+execute it from the second step (after the synthetic dataset have been generated).
+
+```bash
+itwinai exec-pipeline --config config.yaml --pipe-key training_pipeline
+
+# Run from the second step (use python-like slicing syntax).
+# In this case, the dataset is loaded from "data/Image_dataset_synthetic_64x64.pkl"
+itwinai exec-pipeline --config config.yaml --pipe-key training_pipeline --steps 1:
+```
+
+Launch distributed training with SLURM using the dedicated `run.sh` job script:
+
+Train LSTM
+
+```bash
+# Distributed training with torch DistributedDataParallel
+./run.sh config.yaml
+```
+
+Train ConvLSTM
+
+```bash
+# Distributed training with torch DistributedDataParallel
+./run.sh config_conv.yaml
+```
+
+## Running scaling tests
+Scaling tests have been integrated into the eurac usecase to provide timing of experiments run and ths show the power of distributed model training and itwinai. Refer to the following files `runall.sh , scaling-test.sh, torch_dist_final_scaling.py`.
```

Comment:
to :)

Reply:
What exactly are you referring to here? Also, I guess the files have to be changed since `torch_dist_final_scaling.py` isn't really used anymore :)